Github some checks were not successful

github workflow: pyrsia/.github/workflows/demo.yml report error. Details as the below: 
Run apt-get update
Reading package lists...
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)
Error: Process completed with exit code 100.

--- a/.github/workflows/demo.yml
+++ b/.github/workflows/demo.yml
@@ -15,7 +15,6 @@ jobs:
           echo "deb https://pyrsia.github.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
           sudo apt-get update
           sudo apt-get install -y pyrsia
-          sudo chmod 755 /usr/local/bin/pyrsia
         shell: bash
       - name: Run Pyrsia         
         run: |

--- a/.github/workflows/demo.yml
+++ b/.github/workflows/demo.yml
@@ -11,8 +11,8 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y wget gnupg
-          wget -qO - https://pyrsia.github.io/public.key | sudo apt-key add -
-          echo "deb https://pyrsia.github.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
+          wget -qO - https://pyrsia.io/public.key | sudo apt-key add -
+          echo "deb https://pyrsia.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
           sudo apt-get update
           sudo apt-get install -y pyrsia
         shell: bash

--- a/.github/workflows/demo.yml
+++ b/.github/workflows/demo.yml
@@ -8,15 +8,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Run Install
-        run: | 
-          apt-get update
-          apt-get install -y wget gnupg
-          wget -qO - https://pyrsia.github.io/public.key | apt-key add -
-          echo "deb https://pyrsia.github.io/repo bionic main" >> /etc/apt/sources.list
-          apt-get update
-          apt-get install -y pyrsia
-        shell: bash 
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wget gnupg
+          wget -qO - https://pyrsia.github.io/public.key | sudo apt-key add -
+          echo "deb https://pyrsia.github.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
+          sudo apt-get update
+          sudo apt-get install -y pyrsia
+          sudo chmod 755 /usr/local/bin/pyrsia
+        shell: bash
       - name: Run Pyrsia         
         run: |
-          pyrsia --help
+          pyrsia --version
         shell: bash 

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -45,7 +45,7 @@ jobs:
         path: target/debug/pyrsia
         if-no-files-found: error
 
-    # save the DEBIAN control files to be used with the deb packager
+    # save the debian control files to be used with the deb packager
     - uses: actions/upload-artifact@v2
       if: runner.os == 'Linux'
       with:
@@ -84,6 +84,8 @@ jobs:
             mkdir -p pyrsia/usr/local/bin
             cp target/debug/pyrsia_node pyrsia/usr/local/bin
             cp target/debug/pyrsia pyrsia/usr/local/bin
+            chmod 755 pyrsia/usr/local/bin/pyrsia_node
+            chmod 755 pyrsia/usr/local/bin/pyrsia
             echo "Installed-Size: $(echo "$(stat -c%s pyrsia/usr/local/bin/pyrsia) + $(stat -c%s pyrsia/usr/local/bin/pyrsia_node)" | bc)" >> pyrsia/DEBIAN/control
             sed -i.bak "/Version:/s/\$/-$GITHUB_RUN_NUMBER/" pyrsia/DEBIAN/control
             cat pyrsia/DEBIAN/control

